packages/common/util.js

```diff
@@ -169,7 +168,7 @@ exports.renameProperty = (from, to, obj) => {
  * @returns {Object} a shallow clone of the object with `null` and `undefined`
  *   properties removed
  */
-exports.removeNilProperties = (obj) => omitBy(obj, isNil);
+exports.removeNilProperties = (obj) => omitBy(obj, exports.isNil);
```

Comment:
Why not got the other way and use `lodash.isNil` instead of our custom `isNil`? More to the point, why have our own implementation of `isNil` if we're using lodash packages throughout our code?

Reply:
Some of the "isX" functions that lodash provides are somewhat complex, and catch a number of edge cases that we probably wouldn't have thought of. "isNull", "isUndefined", and "isNil" are trivial, though, and I don't think it's worth importing an additional module for those simple cases.

Reply:
You're not kidding: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11973